client/src/components/Header.js

```diff
@@ -10,30 +9,36 @@ function Header() {
         <h1 className="u-text--center">
           <a href="/">Murial's Restaurant</a>
         </h1>
-
-        <section>
-          <Link to="/add">AddFood | </Link>
-          <Link to="/details">Details </Link>
-        </section>
+        <div>
+          <button
+            variant="outline"
+            style={{
+              borderRadius: "90px",
```

Comment:
Does this property accept percentages?  If it does, using `50%` should give you a perfectly round circle.

Reply:
That worked ..thanks